Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -677,3 +677,33 @@
     materialRequirements:
       Steel: 10
       Plasma: 5
+      
+- type: entity
+  id: BoozeDispenserMachineCircuitboard
+  parent: BaseMachineCircuitboard
+  name: booze dispenser machine board
+  description: A machine printed circuit board for a booze dispenser
+  components:
+    - type: Sprite
+      state: service
+    - type: MachineBoard
+      prototype: booze_dispenser
+      requirements:
+        Capacitor: 1
+        Manipulator: 1
+        MatterBin: 1
```

Comment:
just make these use steel + beakers if you aren't going to implement machine upgrading.